homeassistant/components/github/sensor.py

```diff
@@ -121,6 +123,10 @@ def device_state_attributes(self):
             ATTR_OPEN_PULL_REQUESTS: self._pull_request_count,
             ATTR_STARGAZERS: self._stargazers,
         }
+        if self._github_data.latest_release_url is not None:
```

Comment:
This is wrong now. Please revert last commit.

Reply:
my bad, I am so sorry. Copied the code in the wrong. Corrected